www/include/monitoring/status/Services/xml/serviceSummaryXML.php

```diff
@@ -129,21 +122,25 @@
 }
 
 if ($search != "") {
-    $rq1 .= "AND hosts.name like '%" . $search . "%' ";
+    $rq1 .= "AND hosts.name like :search ";
+    $queryValues['search'] = [\PDO::PARAM_STR => '%' . $search . '%'];
 }
 
 if ($instance != -1) {
-    $rq1 .= "AND hosts.instance_id = " . $instance . " ";
+    $rq1 .= "AND hosts.instance_id = :instance ";
+    $queryValues['instance'] = [\PDO::PARAM_INT => $instance];
 }
 
 if ($hostgroups) {
     $rq1 .= " AND hosts.host_id = hg.host_id "
-        . "AND hg.hostgroup_id IN (" . $hostgroups . ") "
+        . "AND hg.hostgroup_id IN (:hostGroup) "
```

Comment:
if only one value is returned from the current "select" filter for host groups you can use the =